.github/workflows/lte-integ-test-bazel-magma-deb.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         env:
           MAGMA_DEV_CPUS: 3
           MAGMA_DEV_MEMORY_MB: 9216
-          MAGMA_PACKAGE: magma_1.8.0_amd64.deb
+          MAGMA_PACKAGE: magma_1.9.0-VERSION-SUFFIX_amd64.deb
```

Comment:
Is `VERSION-SUFFIX` replaced somewhere?

Reply:
This is the "on-the-fly" workflow that will be replaced by using the uploaded artifacts - the uploaded artifacts are not only renamed but also have a correct version in the meta data.